include/config.js

```diff
@@ -219,6 +219,7 @@ Configuration.getBaseConfig = function(multisite) {
         settings: {
             use_memory: true,
             use_cache: false,
+            autoSync: false,
```

Comment:
Since this only applies to plugin settings I'd like to eliminate any ambiguity and make it: ```pb.config.plugins.syncSettingsAtStartup```.  I know it is verbose but hopefully there won't be any mistaking its purpose.